app/models/anime_search.rb

```diff
@@ -0,0 +1,40 @@
+class AnimeSearch
+  include ActiveModel::Model
+
+  SHOW_TYPES = %w(OVA ONA Movie TV Special Music)
+
+  SEASONS = %w(summer fall winter spring)
+
+  attr_accessor(
+    :name,
+    :slug,
+    :show_type,
+    :age_rating,
+    :episode_count,
+    :started_airing,
+    :finished_airing,
+    :genres,
+    :season,
+    :year
+  )
+
+  with_options({allow_blank: true}) do |item|
+    item.validates :show_type, inclusion: SHOW_TYPES
+    item.validates :season, inclusion: SEASONS
+    item.validates :started_airing , :finished_airing, format: { with: /\d{4}-\d{2}-\d{2}/, message: 'date must be formatted as YYYY-MM-DD' }
+    item.validates :year, format: { with: /\d{4}/, message: 'year must be formatted as YYYY' }
+  end
+
+  def apply anime
+
```

Comment:
Shouldn't be space inside the method, but I assume that's intentional.  Also the method parameters should be in parentheses for declarations: `def apply(anime)`